CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/atomist/rug/compare/0.26.1...HEAD
 
+### Fixed
+
+-   Rug Function exceptions are now passed to error handlers.
+    Response now passed to handlers if Rug Function's aren't found
```

Comment:
This sentence makes sense to me after reading the PR but not before  I understood it. Also extra '

Maybe: Response now passed to handlers when unknown Rug Functions are encountered